terraform_compliance/extensions/terraform.py

```diff
@@ -516,6 +518,21 @@ def _add_action_status(self):
             if resource in self.resources:
                 self.resources[resource]['actions'] = resource_change['change']['actions']
 
+    def _add_module_call_source(self):
+        '''
+        Adds module call's source to module's resources as metadata
+        '''
+        for resource in self.resources.values():
+            # removes the for_each signature from addresses
+            # module.a["index_1"].b.c -> module.a.b.c
+            fixed_module_name = '.'.join([word.split('[')[0] for word in resource['address'].split('.')])
```

Comment:
I guess this also works for `count` as well.

Shouldn't this fail if there is no `split` happening here ? Since there wont be any `list`, and `[0]` index of the element should fail.

Reply:
Well, apparently it doesnt fail since you have a test for that already.

Reply:
Conveniently, 'a'.split('b') returns ['a'] instead of []. Adding the `count` tests.